### PR TITLE
Add KTT - KILL THE TAPE - Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN. -- will need to obsolete DC after this gets entered.

### CODE REVIEW

**Comment by palemieux** [comment]
@japierce1028 Why not include the obsolescence step in this pull request?

**Comment by japierce1028** [comment]
Last time I did it as two separate Pull requests and it bounced since the "replaced by" didn't exist. If they are both in the same request would it work? Is that a good idea to have them together?

**Comment by palemieux** [comment]
> Last time I did it as two separate Pull requests and it bounced since the "replaced by" didn't exist. If they are both in the same request would it work? Is that a good idea to have them together? 

It is a good idea to have them together.

**Comment by japierce1028** [comment]
Added the obsolete and it failed, but I can't tell why.

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -1609,7 +1609,11 @@
   },
   {
     "code": "DC",
-    "description": "DIGITAL CUT"
+    "description": "DIGITAL CUT",
+    "obsolete": true,
+    "obsoletedBy": [
+      "KTT"
+    ],
```
Comment (palemieux) [inline src/main/data/facilities.json:1616]:
```suggestion
    ]
```

Extra comma :)

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by palemieux** [comment]
> Added the obsolete and it failed, but I can't tell why.

Extra comma :)

**Comment by japierce1028** [comment]
So many commas, so little time.

**Review by palemieux — APPROVED** [review]

**Pull request merged**